packages/neuron-wallet/src/controllers/export-debug.ts

```diff
@@ -88,6 +89,39 @@ export default class ExportDebugController {
     })
   }
 
+  private addBundledCKBLog = () => {
+    const name = 'bundled-ckb-log.json'
+    const SIZE_TO_READ = 32_000
+
+    return new Promise((resolve, reject) => {
+      const logPath = path.resolve(ckbPath(), 'data', 'logs', 'run.log')
+      if (!fs.existsSync(logPath)) { return reject(new Error("File not found")) }
+
+      const fileStats = fs.statSync(logPath)
+      const position = fileStats.size - SIZE_TO_READ
+
+      fs.open(logPath, 'r', (openErr, fd) => {
+        if (openErr) { return reject(openErr) }
+        fs.read(fd, Buffer.alloc(SIZE_TO_READ), 0, SIZE_TO_READ, position, (readErr, _, buffer) => {
+          if (readErr) {
+            reject(readErr)
+          } else {
+            resolve(buffer.toString('utf8'))
+          }
+          fs.close(fd, closeErr => {
```

Comment:
Thanks for the updates.
I think `fs.close` may be better to be called within the `then` and `catch` of this promise, because it might not be able to reach this point to close the `fd` due to potential errors above within the `fs.open` callback.